Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -20,26 +25,80 @@ project which uses the Legacy XR pipeline.
 
 1. Ensure that [developer mode](https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is enabled on your device. Installing the Oculus ADB Drivers is optional.
 
-1. Adding the Oculus XRSDK Data Provider
+## Setting up the scene
     - Create a new Unity scene or open a pre-existing scene like HandInteractionExamples
+    
     - Add MRTK to the scene by navigating to **Mixed Reality Toolkit** > **Add to Scene and Configure**
-    - Select the MixedRealityToolkit game object in the hierarchy and select **Copy and Customize** to clone the default mixed reality profile.
 
-    ![CloneProfile](../Images/CrossPlatform/CloneProfile.png)
+## Using the Oculus XRSDK Data Provider
+
+1. Configure your profile to use the **Oculus XRSDK Data Provider**
+    - If not intending to modify the configuration profiles
+        - Change your profile to DefaultXRSDKInputSystemProfile and go to **Build and deploy your project to Oculus Quest**
```

Comment:
A link to the **Build and deploy your project to Oculus Quest** section would be helpful here
```suggestion
        - Change your profile to DefaultXRSDKInputSystemProfile and go to [Build and deploy your project to Oculus Quest](OculusQuestMRTK.md#build-and-deploy-your-project-to-oculus-quest)
```